lib/src/util.dart

```diff
@@ -157,7 +157,6 @@ bool isDartFile(String filePath) => path.extension(filePath) == '.dart';
 ///     // False
 bool pathLooksLikeCode(String filePath) =>
     !filePath.contains('/.') &&
-    !filePath.contains('dev.dart') &&
```

Comment:
The `isDart2` codemod should not run on `dev.dart` will it after this change?

Reply:
If that's the case, we should update the `isDart2` codemod to do its own skipping of `dev.dart` files

Reply:
https://github.com/Workiva/dart2_upgrade_helper/blob/master/bin/d1down_remove_dart_version.dart#L29 looks like it does already.